packit_service/utils.py

```diff
@@ -65,3 +67,41 @@ def load_job_config(job_config: JobConfig):
 
 def dump_job_config(job_config: JobConfig):
     return JobConfigSchema().dump_config(job_config) if job_config else None
+
+
+def pretty_time(time: datetime) -> Optional[str]:
+    """
+    Get a pretty string like 'an hour ago', 'yesterday', '3 months ago',
+    'just now'.
+    https://stackoverflow.com/questions/1551382/user-friendly-time-format-in-python
+    """
```

Comment:
Nice improvement!

I am only wondering if we can't use some library for that. After (very quick) search, [arrow](https://arrow.readthedocs.io/en/latest/) should be packaged for Fedora.

Reply:
I would argue if it's worth replacing a single function with a library

Reply:
That's a question, but
- this looks too manual and too reinvent-the-wheel to me,
- it's already in Fedora what I am using as some quality certificate,
- we might find also other features useful (timezone support, locales, easier shifts, ...).

I am wondering if flask does not have something like this already there?

Reply:
yes, those are good points, my intention is that when we add a new dependency, our images grow and we start relaying on the new library and all of its dependencies - in this case I'd say it's "cheaper" to go with the function for now and use a library once we realize we need more - which we probably will in the dashboard when we replace the build result page

luckily, this is p-s, we're not packaging it as an rpm, so we can do w/e we want

Reply:
As I read the pros and cons, I would go for now with this simple function and not install the library for this functionality. If we will extend the functionality in the future, we can do it then.